### ISSUE TITLE
Create A Landing Page

### ISSUE THREAD

**Issue by yosefalnajjarofficial** [ca-g7/code-wizards#3]
Create the HTML and CSS styles for the homepage which the user first see when entering a site
1- It should look satisfying and easy convenient 
2- Beautiful design and a unique one
3- It should be readable for screen readers
